freeipa/src/main/java/com/sequenceiq/freeipa/sync/StackStatusCheckerJob.java

```diff
@@ -105,23 +105,28 @@ public void syncAStack(Stack stack) {
                     if (!checkableInstances.isEmpty()) {
                         SyncResult syncResult = freeipaChecker.getStatus(stack, checkableInstances);
                         if (DetailedStackStatus.AVAILABLE == syncResult.getStatus()) {
-                            for (Map.Entry<InstanceMetaData, DetailedStackStatus> entry : syncResult.getInstanceStatusMap().entrySet()) {
-                                updateInstanceStatus(entry.getKey(), entry.getValue());
-                            }
-                            updateStackStatus(stack, syncResult, null, alreadyDeletedCount);
+                            String instanceStatusInfo = updateInstanceStatus(syncResult.getInstanceStatusMap());
+                            String freeipaStatusInfo = updateStackStatus(stack, syncResult, null, alreadyDeletedCount);
+                            LOGGER.info(":::Auto sync::: freeipa status from healtch check: " + freeipaStatusInfo + instanceStatusInfo);
```

Comment:
I am not sure what log level we should have for this. If debug output is enabled in prod, then it probably should be debug level.

Reply:
Thanks! I changed to debug level.